manual_user_install.sh

```diff
@@ -0,0 +1,265 @@
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-phal-plugin-dashboard
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-phal-plugin-alsa
+
+    # install required skills
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-volume
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-fallback-unknown
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-stop
+    pip3 install git+https://github.com/OpenVoiceOS/skill-alerts
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-personal
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-naptime
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-date-time
+
+    # You can uncomment these lines if the deprecation notices are flooding your logs or slowing the system
+    #sed -i '/\@deprecated/d' $HOME/.local/lib/python3.9/site-packages/ovos_utils/fingerprinting.py
+    #sed -i '/\@deprecated/d' $HOME/.local/lib/python3.9/site-packages/ovos_utils/configuration.py
+    #sed -i '/\@deprecated/d' $HOME/.local/lib/python3.9/site-packages/ovos_utils/ovos_service_api.py
+    #sed -i '/\@deprecated/d' $HOME/.local/lib/python3.9/site-packages/ovos_utils/signal.py
+
+    if [[ ! -d $HOME/.config/mycroft ]]; then
+        mkdir -p $HOME/.config/mycroft
+    fi
+    cp $SCRIPT_DIR/stage-core/01-ovos-core/files/mycroft.conf $HOME/.config/mycroft/
+    echo $SCRIPT_DIR
+    echo
+    echo "Done installing OVOS core"
+    }
+
+function install_systemd (){
+    echo
+    echo "Installing systemd files"
+    echo
+
+    # install the hook files
+    cp $SCRIPT_DIR/stage-core/01-ovos-core/files/ovos-systemd-skills $HOME/.local/bin/
+    cp $SCRIPT_DIR/stage-core/02-messagebus/files/ovos-systemd-messagebus $HOME/.local/bin/
+    cp $SCRIPT_DIR/stage-audio/01-speech/files/ovos-systemd-audio $HOME/.local/bin/
+    cp $SCRIPT_DIR/stage-audio/02-voice/files/ovos-systemd-dinkum-listener $HOME/.local/bin/
+    cp $SCRIPT_DIR/stage-phal/01-user/files/ovos-systemd-phal $HOME/.local/bin/
+    cp $SCRIPT_DIR/stage-phal/02-admin/files/ovos-systemd-admin-phal $HOME/.local/bin
+
+    chmod +x $HOME/.local/bin/ovos-systemd*
+
+    # sdnotify is required
+    pip3 install sdnotify
+
+    # install the service files
+    if [[ ! -d $HOME/.config/systemd/user ]]; then
+        mkdir -p $HOME/.config/systemd/user
+    fi
+    cp $SCRIPT_DIR/stage-core/01-ovos-core/files/ovos.service $HOME/.config/systemd/user/
+    cp $SCRIPT_DIR/stage-core/01-ovos-core/files/ovos-skills.service $HOME/.config/systemd/user/
+    cp $SCRIPT_DIR/stage-core/02-messagebus/files/ovos-messagebus.service $HOME/.config/systemd/user/
+    cp $SCRIPT_DIR/stage-audio/01-speech/files/ovos-audio.service $HOME/.config/systemd/user/
+    cp $SCRIPT_DIR/stage-audio/02-voice/files/ovos-dinkum-listener.service $HOME/.config/systemd/user/
+    cp $SCRIPT_DIR/stage-phal/01-user/files/ovos-phal.service $HOME/.config/systemd/user/
```

Comment:
In order for this to work correctly, it needs to run as a system service. eg. it will use `sudo`

Reply:
I'm not sure what you mean by the words "this" and "it".  Do you mean run the services as 'systemctl' rather than 'systemctl --user' services? Or something else?  I'm working right at the edge of what I understand and learning everything else as I go along. Thanks.

Reply:
Sorry, `ovos-admin-phal` because it needs sudo privileges

Reply:
Got it. Thanks!